test/lint/sorted_sets.rb

```diff
@@ -293,6 +293,15 @@ def test_zscore
       assert_equal(+Float::INFINITY, r.zscore("bar", "s2"))
     end
 
+    def test_zmscore
+      r.zadd "foo", [1, "s1", 2, "s2"]
+
+      assert_equal [1.0, 2.0], r.zmscore("foo", ["s1", "s2"])
```

Comment:
I don't get it. You defined a variadic method, but here you pass an array.

It should be `r.zmscore("foo", "s1", "s2")`